templates/default/clients.conf.erb

```diff
@@ -6,9 +6,8 @@
 
 <% node['freeradius']['clients'].each do |client, client_hash| %>
 client <%= client %> {
-  ipaddr = <%= client_hash['ipaddr']  %>
-  netmask = <%= client_hash['netmask'] %>
-  secret = <%= client_hash['secret'] %>
-  nastype = <%= client_hash['nastype'] %>
+<% client_hash.each do |key, val| %>
```

Comment:
From a security perspective, do you think it makes sense to create a constant of white-listed attributes and assign based on that? I'm not overly opinionated on the matter, but figure it is worth mentioning.